feat: Add curve query support for MSO2KB series scopes (#269)
tektronix/tm_devices: CHANGELOG.md

## Unreleased

> - feat: Added basic command support for MSO2KB series scopes ([#269](https://github.com/tektronix/tm_devices/pull/269))

Reviewer: ```suggestion
### Added

- Added curve query support for the MSO2KB series scopes
```

## Unreleased

> ### Added

Reviewer: ```suggestion
Things to be included in the next release go here.

### Added
```

This is required for the release automation

Author: "Things.." text added back in.

> ### Added

Reviewer: ```suggestion
Things to be included in the next release go here.

### Added
```

Things to be included in the next release go here.

### Added

- Added curve query support for the MSO2KB series scopes

---

> ## v2.2.2 (2024-08-14)

Reviewer: It looks like this was added on accident.

Reviewer: This shows you adding the v2.2.2 changes, but they are already in the file. The only changes that should show up are the ones in the unreleased section. You need to delete the duplicate sections. It should be obvious if you look in the file. There are duplicate sections that should not be there.

Reviewer: Everything from line 23 to line 83 needs to be removed. It is duplicating the sections that come next.
